src/species.jl

```diff
@@ -1,54 +0,0 @@
-"""
```

Comment:
This is what I meant - even in this view, it's going to tell me that all of species.jl got removed and all of taxon.jl got written, so I can't see what actually happened in the update.